stdlib/2and3/builtins.pyi

```diff
@@ -190,10 +190,10 @@ class int:
 
     def __eq__(self, x: object) -> bool: ...
     def __ne__(self, x: object) -> bool: ...
-    def __lt__(self, x: int) -> bool: ...
-    def __le__(self, x: int) -> bool: ...
-    def __gt__(self, x: int) -> bool: ...
-    def __ge__(self, x: int) -> bool: ...
+    def __lt__(self, x: Union[int, float]) -> bool: ...
+    def __le__(self, x: Union[int, float]) -> bool: ...
+    def __gt__(self, x: Union[int, float]) -> bool: ...
+    def __ge__(self, x: Union[int, float]) -> bool: ...
```

Comment:
This would actually be a lie:

```python
>>> x = 3
>>> x.__lt__(4.3)
NotImplemented
```

When writing `3 < 4.3` Python actually calls `float.__gt__()` after `int.__lt__()` returns `NotImplemented`. This behavior should be replicated by type checkers.

Reply:
Good point. Thanks.